addon_service/common/known_imps.py

```diff
@@ -73,6 +77,8 @@ class AddonImpNumbers(enum.Enum):
     BOX_DOT_COM = 1001
     ZOTERO_ORG = 1002
     S3 = 1003
+    GOOGLE_DRIVE = 1004
```

Comment:
I know we have this number for **Mendeley**, I think you can just change this to `1005`, or/and Figshare to `1007`
`MENDELEY = 1004`

Reply:
This is Google drive related